src/styles/custom.css

```diff
@@ -0,0 +1,4 @@
+/* Fix for tooltips breaking layout on mobile devices (<526px width). */
+
+
+@media (max-width: 526px) {#gas-info, #transaction-info, #message-info{max-width: calc(100% - 10px);}}
```

Comment:
Please use tailwind classes instead of creating a new css file.
If you need to do specific amounts like 526px, you can use a class like `max-w-[526px]`, but it would be better if you can make this work with just the `max-w` classes in tailwind's base set.

Reply:
Thanks for the suggestion! Unfortunately, this appears to be a limitation or bug in the Tooltip library. When we try to pass a className like max-w-[calc(100%-10px)] sm:max-w-[526px], it gets applied to an unexpected <a> tag instead of the actual tooltip. This behavior prevents us from fully leveraging Tailwind classes in this case. For now, me had to rely on a custom CSS file to address this issue.

Reply:
There is a property inside the Tooltip called `data-tooltip-class-name` which directly modifies the `className` for the tooltip component. I vote we also move this part to the widgets @jmrossy

Reply:
@NikolaiKryshnev would you like to give this classname-based approach a try? Also from the CI failure it looks like you need to run `yarn prettier` to fix the formatting issue